packages/react/src/components/cookieConsentCore/monitorAndCleanBrowserStorages.js

```diff
@@ -297,9 +285,9 @@ export default class MonitorAncCleanBrowserStorages {
    * @param {number} [monitorInterval=500] - The interval in milliseconds at which to monitor the stored keys.
    * @param {boolean} [remove=false] - Indicates whether to remove the stored keys or not.
    */
-  init(cookieHandler, monitorInterval = 0, remove = false) {
+  init(cookieHandler, monitorInterval = 500, remove = false) {
```

Comment:
This enables the monitoring by default. Should it still be "0" by default?

Reply:
Hmmm I actually don't know should or shouldn't it be since all the JSONs set the monitorInterval and the default value as documented right above says it's 500?

Reply:
Although it won't remove anything if remove = false (default)

Reply:
Ok, both ways are good, as long as it is not 50ms. We'll see how many complain about tons of console.log warnings. 😄